packages/core/admin/admin/src/pages/SettingsPage/pages/Webhooks/ListView/index.js

```diff
@@ -473,8 +387,8 @@ const ListView = () => {
       </Main>
       <ConfirmDialog
         isOpen={showModal}
-        onToggleDialog={handleToggleModal}
-        onConfirm={handleConfirmDelete}
+        onToggleDialog={() => setShowModal((prev) => !prev)}
+        onConfirm={confirmDelete}
```

Comment:
Could you also set `isConfirmButtonLoading` please, when a mutation is running? I think that is an important user feedback :)